persistence/persistence-api/src/main/resources/ehcache-disk-only.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It seems to me we could just have one ehcache.xml configuration file. Can we inject if/else conditions in the xml files that controlled by properties like `turn.on.disk.cache` and `turn.on.heap.cache`? This could avoid some duplications in the configuration. From what I see, they are almost identical.

Reply:
There is probably a way to do that. We can look into it.

Reply:
done